### ISSUE TITLE
Clicking canvas does not create nodes

### ISSUE THREAD

**Issue by StrungSilver** [makyo/polycul.es#25]
Perhaps it's not designed to work with Windows desktop PCs, but using the mouse to try to create new nodes does not function at all. It works (too well) on a tablet or smartphone, to the point of making it difficult or impossible to edit a node - during the same session - after it's been initially created.

**Comment by makyo** [comment]
Oh, hmm! Can you provide browser and version? Not at a computer not, but
I'll hunt down a Windows box at some point to figure this out (works in
Firefox and chrome, here).

Thanks!

**Comment by fabianhjr** [comment]
Can replicate in Linux-Firefox, will try to get a look on the weekend

```
 ~  uname -r
4.12.6-1-ARCH
 ~  yaourt -Qi firefox
Name            : firefox
Version         : 55.0.1-1
```

**Comment by makyo** [comment]
Fixed by 75b2b98
